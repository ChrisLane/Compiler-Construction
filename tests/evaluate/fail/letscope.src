// This test is to make sure that a let variable cannot be accessed outside of scope. This should not evaluate.
main() {
    let x = 50 {
        x;
    };

    print x;
}
