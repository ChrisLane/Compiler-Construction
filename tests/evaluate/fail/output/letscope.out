Syntax error at line 6 and column 4: print.
Fatal error: exception Error.Error
